tests/Unit/AcceleratorViewCopy/avstress_0x18.cpp

```diff
@@ -68,25 +68,33 @@ int main(){
         }
     }
 
-    if (testsToRun & 0x8) {
+// Create a vector of hc::completion_future for async copy synconization
+    std::vector<hc::completion_future> cfs;
+    
+    if (testsToRun & 0x18) {
```

Comment:
Could we use more meaningful names instead of an hex value?  
For example, if (testsToRun & HostToDeviceAsyncCopyTest) ?